crates/full-node/sov-db/src/state_db_nomt.rs

```diff
@@ -70,6 +70,20 @@ impl<H: digest::Digest<OutputSize = digest::typenum::U32> + Send + Sync> NomtSta
             CommitStatus::Completed => {
                 tracing::trace!("Commit flag is `Completed`, proceeding as usual");
             }
+            CommitStatus::Unknown => {
```

Comment:
Intuitively this seems like it should be an error to me. Why is it that can't we distinguish between the file having gotten corrupted and the file not having existed previously?